index.md

```diff
@@ -0,0 +1,94 @@
+# Workshop Wise Index
+
+## Table of Contents
+
+1. [Golang Workshop](#golang-workshop)
+1. [GSoc IntroSem](#gsoc-introsem)
+1. [Git and Github Workshop](#git-github-workshop)
+1. [Lightning Talks](#lightning-talks)
+1. [KOSS Introsem](#koss-introsem)
+1. [Linux Install Fest](#gnulinux-installation)
+1. [Linux Shell and VIM](#linux-shell-and-vim)
+1. [Python Workshop](#python-workshop)
+1. [Rest APIs](#rest-apis)
+1. [Hacking](#hacking-workshop)
+1. [Javascript Workshop](#javascript)
+
+## Golang Workshop
+
+* **2024**
+  * [**Spring**](./2024/Spring/golang-workshop)
+* **2018**
+  * [**Autumn**](./2018/Autumn/golang)
+
+
+## GSoC IntroSem
+
+* **2024**
+  * [**Spring**](./2024/Spring/GSoC-Intro-Sem)
+* **2022**
+  * [**Autumn**](./2022/Autumn/GSoC-Intro-Seminar)
+* **2021**
+  * [**Autumn**](./2021/Autumn/GSoC-Intro-Sem)
+* **2019**
+  * [**Autumn**](./2019/Autumn/GSoC-Intro-Sem)
+
+## Git-GitHub Workshop
+
+* **2023**
+  * [**Autumn**](./2023/Autumn/Git-github)
+  * [**Spring**](./2023/Spring/Git-github)
+* **2022**
+  * [**Autumn**](./2022/Autumn/Git-github)
+* **2021** 
+  * [**Autumn**](./2021/Autumn/Git-github)
+* **2020**
+  * [**Autumn**](./2020/Autumn/git-github)
+* **2018**
+  * [**Autumn**](./2018/Autumn/git-github)
+
+## Lightning Talks
+
+* **2023**
+  * [**Autumn**](./2023/Autumn/Lightning-Talks)
+* **2022**
+  * [**Autumn**](./2022/Autumn/Lightning-Talks)
+* **2021**
+  * [**Autumn**](./2021/Autumn/Lightning-Talks)
+
+## KOSS IntroSem
+
+* **2023**
+  * [**Spring**](./2023/Spring/KOSS-Intro-Sem)
+* **2021**
+  * [**Autumn**](./2021/Autumn/KOSS-Intro-Sem)
+
+## GNU/Linux Installation
+
+* **2023**
+  * [**Spring**](./2023/Spring/Ubuntu-Install-Fest)
+
+## Linux Shell and VIM
+
+* **2021**
+  * [**Spring**](./2021/Spring/Linux-Shell-and-VIM)
+
+## Python Workshop
+
+* **2021** 
+  * [**Spring**](./2021/Spring/Python-Basic-&-Advanced)
+
+## Rest APIs
```

Comment:
This event could possibly be clubbed under the "Python" section. Alternatively, such one-off events can be kept under a "Miscellaneous" section.

Reply:
made a misc section